assets/css/app.css

```diff
@@ -272,3 +272,15 @@ ul.sidebar .sidebar-list a.active {
   border-left: 3px solid #fff;
   background: #2d3e63;
 }
+
+.sortable {
+  cursor: pointer;
+}
+
+.st-sort-ascent .glyphicon:before {
```

Comment:
Perfect !
I was prefering font-awesome also, but I didn't noticed it was already there.